src/main/java/org/candlepin/subscriptions/resource/TallyResource.java

```diff
@@ -305,6 +339,31 @@ private Set<HardwareMeasurementType> getContributingTypes(ReportCategory categor
     return contributingTypes;
   }
 
+  private List<BillableUsageRemittanceEntity> getRemittances(
+      ReportCriteria reportCriteria, BillingCategory billingCategory, String billingAcctId) {
```

Comment:
We have a `billingAccountId` field in `ReportCriteria`. Please use that instead. Also, I'd like to see `billingCategory` added as a field to `ReportCriteria`.